EmployeeInfo.java

```diff
@@ -0,0 +1,69 @@
+package tasks;
+
+/**
+ * This program demonstrates encapsulation and get and sets the employee's information 
+ * 
+ * @author Jayesh Soni
+ * @since 2024-04-22
+ */
+
+/**
```

Comment:
Above the Class only one documentation comment or anytime Need to add only one documentation comment All thing you can add in that. You also can you <br> Tag for the new line and another some html tag also